src/sentry/sentry_metrics/indexer/limiters/cardinality.py

```diff
@@ -49,7 +49,15 @@ def _construct_quotas(use_case_id: UseCaseID) -> Optional[Quota]:
     """
 
     if use_case_id in USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS:
-        quota_args = options.get(USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS[use_case_id])
+        custom_card_limit = options.get(USE_CASE_ID_CARDINALITY_LIMIT_QUOTA_OPTIONS[use_case_id])
+
+        # check that the option is set to something
+        if len(custom_card_limit) > 0:
+            quota_args = custom_card_limit
+        else:
+            quota_args = options.get(
+                "sentry-metrics.cardinality-limiter.limits.generic-metrics.per-org"
```

Comment:
Nit: Lets define a constant for the default option to make code readability easier.